app/templates/components/submission-status-cell.hbs

```diff
@@ -19,33 +20,34 @@
 </style>
 
 <div class="text-left">
-    {{#if (eq record.source 'other')}}
-      {{#if (eq record.submitted true)}}
-      <span class="status accepted pr-1">
+  <span class="status {{status}} pr-1">
+    {{#if (eq status "Complete")}}
+      {{#if (eq record.source "other")}}
         <span tooltip-position="left" tooltip="Funder has receive a submission for this publication from a different source.">
-          <i class="fas fa-info-circle d-inline"></i> Submitted
+          <i class="fas fa-info-circle"></i> {{status}}
         </span>
-      </span>
       {{else}}
-        <span tooltip-position="left" tooltip="Your funder is aware of this publication and is expecting the deposit of your manuscript.">
-          <i class="fas fa-info-circle d-inline"></i>  Manuscript expected
-        </span>
-        {{!-- <span class="status not-started pr-1">
-          <i class="fas fa-circle"></i> Manuscript expected
-        </span> --}}
+        <i class="fas fa-circle"></i> {{status}}
       {{/if}}
-    {{else}}
-      <span class="status {{record.aggregatedDepositStatus}} pr-1">
-        {{#if (eq status "Accepted")}}<i class="fas fa-circle"></i> {{status}}{{/if}}
-        {{#if (eq status "In Progress")}}<i class="fas fa-circle"></i> {{status}}{{/if}}
-        {{#if (eq status "Stalled")}}<i class="fas fa-circle"></i> {{status}} (<a href="#">?</a>){{/if}}
-        {{#if (eq status "Not Started")}}
+    {{/if}}
+    {{#if (eq status "Manuscript expected")}}
+      <span tooltip-position="left" tooltip="Your funder is aware of this publication and is expecting the deposit of your manuscript.">
+        <i class="fas fa-info-circle"></i> {{status}}
+      </span>
+    {{/if}}
+    {{#if (eq status "Submitted")}}
+      {{#if (eq record.source "pass")}}
         <span tooltip-position="left" tooltip="This submission is in queue to be transmitted to target its repositories.">
```

Comment:
Change to "This submission has either been sent to all target repositories, or is in the queue and will be submitted shortly"

Reply:
this looks good.